components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -564,9 +560,13 @@ public void testFilteringUsersWithGET(List<org.wso2.carbon.user.core.common.User
             node = filterTreeManager.buildTree();
         }
 
-        List<Object> result = scimUserManager.listUsersWithGET(node, 1, null, null, null, null,
-                requiredClaimsMap);
-        assertEquals(expectedResultCount, result.size());
+        try {
+            List<Object> result = scimUserManager.listUsersWithGET(node, 1, null, sortBy, sortOrder, null,
+                    requiredClaimsMap);
+            assertEquals(expectedResultCount, result.size());
+        } catch (NotImplementedException e){
+            System.out.println("Sorting is not supported");
```

Comment:
Why do we need this System.out.println here, can't we have something as follows to assert the exceptions

@Test(expected = NotImplementedException.class)

Reply:
Let's add an assertion,  "System.out.println" is not a best prctice to use

Reply:
Resolved from [7326618](https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/732661849be885ed199390ee3a12e57bd38181b7)